examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,109 @@
+# basis function* (RBF) kernel:
+#
+# .. math::
+#     K_{i,j} = e^{-\|\bf{r}_i-\bf{r}_j\|^2/2\sigma^2},
+#
+# where :math:`\bf{r}_i` are the coordinates of point :math:`i` and :math:`\sigma`
+# is a kernel parameter that determines the scale of the kernel. Points that are much further
+# than a distance :math:`\sigma` from each other lead to small entries of the kernel matrix,
+# whereas points much closer than :math:`\sigma` generate large entries. The parameter
+# :math:`\sigma` is set to 1.0 in this example. For kernel matrices that are 
+# positive-semidefinite, there exist efficient quantum-inspired classical algorithms
+# for permanental point process sampling :cite:`jahangiri2019point`. In this tutorial
+# we restrict ourselves to such positive-semidefinite kernels and we use the quantum-inspired
+# classical algorithm for generating the permanental point process samples.
+
+K = points.kernel(R, 1.0)
+
+##############################################################################
+# We generate 10 samples with an average number of 50 points per sample by calling
+# the :func:`~.points.sample` function of the :mod:`~.gbs.points` module.
+
+samples = points.sample(K, 50.0, 10)
+
+##############################################################################
+# We visualize the first sample by using the :func:`~.plot_points` function of 
+# the :mod:`~.gbs.plot` module.
+
+import plotly
+#plotly.offline.plot(plot_points(R, samples[0]), filename="Points.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/Points.html
+
+##############################################################################
+# Outlier Detection
+# -----------------
+#
+# When the distribution of points in a given space is inhomogenous, GBS point processes typically
+# sample points from the dense regions. This feature can be used to detect high-density outlier 
+# points in a data set. In this example, we create two dense clusters and place them in a 
+# two-dimensional space containing evenly-distributed points. The GBS point process samples points 
+# from the dense clusters, with a higher probability.
+#
+# We first create the data points.
+
+from sklearn.datasets import make_blobs
+R = np.concatenate((make_blobs(n_samples=50, centers=[[5, 5], [15, 15]], cluster_std=1)[0],
+                    np.array([(i, j) for i in range(20) for j in range(20)])))
+
+##############################################################################
+# Then construct the kernel matrix.
+
+K = points.kernel(R, 1.0)
+
+##############################################################################
+# And generate 500 samples.
+
+samples = points.sample(K, 50.0, 500)
+
+##############################################################################
+# We obtain the indices of 50 points that appear most frequently in the GBS point
```

Comment:
Again, you can combine this with the lines below so there is a better flow to the tutorial